### PR TITLE
feat-added home page

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Home Page

### CODE REVIEW

File: pages/home.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			/></a>
+			<div class="nav__input">
+				<input class="input" placeholder="Search For items" type="te" />
+				<i class="fa fa-search search__icon"></i>
+			</div>
+
+			<div class="nav__links">
+				<div class="nav__login">
+					<p>Hello User</p>
+					<button class="nav__login-btn">Login</button>
```
Comment (Prabhav-Sharma) [inline pages/home.html:31]: The cursor remains the same on hovering, might be nice if it changed to pointer on button:hover

File: layout/header.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	color: var(--background-color);
+}
+.nav__login-btn {
+	padding: 2px 8px;
+	font-size: 1.2rem;
+	border-radius: var(--p4);
+	border: 1px solid var(--primary-color);
+	color: var(--secondary-color);
+	background-color: var(--background-color);
+}
```
Comment (amarphule) [inline layout/header.css]: You could add `cursor:pointer`

**Review by amarphule — CHANGES_REQUESTED** [review]

**Review by abhijeetekad — APPROVED** [review]

File: pages/home.html (hunk 1 above)
Comment (prathmeshjagtap) [reply pages/home.html:31]: Did the required  changes

File: layout/header.css (hunk 2 above)
Comment (prathmeshjagtap) [reply layout/header.css]: Added the required  changes

**Review by amarphule — APPROVED** [review]

**Pull request merged**